homeassistant/components/fritzbox/sensor.py

```diff
@@ -29,9 +34,58 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
             entities.append(FritzBoxTempSensor(device, fritz))
             devices.add(device.ain)
 
+        if device.battery_level is not None:
+            entities.append(FritzBoxBatterySensor(device, fritz))
+            devices.add(f"{device.ain}_{DEVICE_CLASS_BATTERY}")
```

Comment:
```suggestion
            devices.add(f"{device.ain}_battery")
```

The unique id shouldn't contain the device class since it could change name in the future

Reply:
Thanks for that hint :+1: